src/main/java/app/bpartners/geojobs/service/event/AutoTaskStatisticRecomputingSubmittedService.java

```diff
@@ -47,38 +42,33 @@ public AutoTaskStatisticRecomputingSubmittedService(
   @Override
   public void accept(AutoTaskStatisticRecomputingSubmitted event) {
     var jobId = event.getJobId();
-    var lastDurationValue = event.getMaxConsumerBackoffBetweenRetriesDurationValue();
     var attemptNb = event.getAttemptNb();
     var optionalTiling = tilingJobRepository.findById(jobId);
     if (optionalTiling.isEmpty()) {
       var zoneDetectionJob =
           zoneDetectionJobRepository
               .findById(jobId)
               .orElseThrow(() -> new NotFoundException("Job.id=" + jobId + " not found"));
-      processJob(zoneDetectionJob, jobId, lastDurationValue, attemptNb);
+      processJob(zoneDetectionJob, jobId, attemptNb);
     } else {
       var tilingJob = optionalTiling.get();
-      processJob(tilingJob, jobId, lastDurationValue, attemptNb);
+      processJob(tilingJob, jobId, attemptNb);
     }
   }
 
-  private void processJob(Job job, String jobId, long lastDurationValue, int attemptNb) {
+  private void processJob(Job job, String jobId, int attemptNb) {
     var taskStatisticRecomputingSubmitted = new TaskStatisticRecomputingSubmitted(jobId);
     if (job instanceof ZoneDetectionJob) {
       zoneDetectionStatisticConsumer.accept(taskStatisticRecomputingSubmitted);
     } else if (job instanceof ZoneTilingJob) {
       tilingStatisticConsumer.accept(taskStatisticRecomputingSubmitted);
     }
-
-    if (!job.isFailed() && !job.isSucceeded()) {
-      if (attemptNb < ATTEMPT_FOR_256_MINUTES_DURATION) {
-        long maxConsumerBackoffBetweenRetriesDurationValue = lastDurationValue * 2;
-        int newAttemptNb = attemptNb + 1;
-        eventProducer.accept(
-            List.of(
-                new AutoTaskStatisticRecomputingSubmitted(
-                    jobId, maxConsumerBackoffBetweenRetriesDurationValue, newAttemptNb)));
-      }
+    if (job.isFinished()) {
+      return;
+    }
+    if (attemptNb > ATTEMPT_FOR_256_MINUTES_DURATION) {
+      return;
     }
+    throw new RuntimeException("Fail on purpose so that message is not ack, causing retry");
```

Comment:
As the structure given, the task statistic compute must be done before the exception here.